lib/src/entrypoint.dart

```diff
@@ -246,13 +246,15 @@ class Entrypoint {
 
     if (dryRun) {
       result.summarizeChanges(type, dryRun: dryRun);
+      if (type == SolveType.UPGRADE) result.showOutdated(type);
```

Comment:
One more thing I just noticed.

We should move the conditional into `SolveResult.summarizeChanges`, then we avoid repeating it.

Reply:
Updated.

I also thought this initially but didn't implement because I thought, `result.showOutdated` could be used for some other `type` in future.